db/c.cc

```diff
@@ -3960,10 +3960,12 @@ int rocksdb_options_get_wal_compression(rocksdb_options_t* opt) {
 
 rocksdb_ratelimiter_t* rocksdb_ratelimiter_create(int64_t rate_bytes_per_sec,
                                                   int64_t refill_period_us,
-                                                  int32_t fairness) {
+                                                  int32_t fairness,
+                                                  bool auto_tuned) {
   rocksdb_ratelimiter_t* rate_limiter = new rocksdb_ratelimiter_t;
   rate_limiter->rep.reset(
-      NewGenericRateLimiter(rate_bytes_per_sec, refill_period_us, fairness));
+      NewGenericRateLimiter(rate_bytes_per_sec, refill_period_us, fairness,
+                            RateLimiter::Mode::kWritesOnly, auto_tuned));
```

Comment:
If exposing `RateLimiter::Mode` is something of interest.  I can expose it in the C API in a separate PR.